__tests__/app.test.js

```diff
@@ -39,4 +40,25 @@ describe('GET: /api/categories', () => {
             .expect(404)
 
     })
+})
+
+describe('GET: /api', () => {
+    it('responds with correct status code 200', () => {
+        return request(app)
+            .get('/api')
+            .expect(200)
+    })
+    it('gives an object detailing the endpoints available', () => {
+        return request(app)
+            .get('/api')
+            .then((res) => {
+                const body = res.body.endpoints
+                expect(body).toBeInstanceOf(Object)
+                for (const prop in body) {
+                    expect(body[prop]).toHaveProperty('description');
+                    expect(body[prop]).toHaveProperty('queries'); expect(body[prop]).toHaveProperty('exampleResponse');
+                }
```

Comment:
You could also test for the values' types.

Reply:
Maybe have a look at [`.toMatchObject( )`](https://jestjs.io/docs/expect#tomatchobjectobject) and [`expect.objectContaining( )` ](https://jestjs.io/docs/expect#expectobjectcontainingobject)